mozilla/bedrock
# Configure analytics for Relay product pages #13322

## One-line summary

Configure analytics for Relay product pages #13322

## Significant changes and points to review

- add analytics to product settings
- add CTAs to buttons
- add js for tracking begin_checkout and attribution

## Issue / Bugzilla link

#13322

## Testing

https://www-demo5.allizom.org/en-US/products/relay/

In [GTM](https://tagmanager.google.com/#/container/accounts/52671733/containers/1075029/workspaces/1000220):

- Click "preview" to open the tag assistant website.
- Add the domain "https://www-demo5.allizom.org/en-US/products/relay/" and connect to it
- Disable add-ons and toggle off DNT, etc until tag assistant indicates it has a connection
- Click things on the Relay site that should have tracking on them
- Return to the tag assistant window and look for the CTA Clicks and Begin Checkout events

Check:

- [x] All buttons fire a cta click event for GA3
- [x] Waitlist signups fire a `newsletter subscription` event for GA3
- [x] begin_checkout event is fired for GA4 for any _paid_ subscription referral to FxA
